sdk/storage_blobs/src/container/operations/list_blobs.rs

```diff
@@ -23,6 +23,7 @@ operation! {
     ?include_deleted: bool,
     ?include_tags: bool,
     ?include_versions: bool,
+    ?initial_marker: NextMarker,
```

Comment:
```suggestion
    ?marker: NextMarker,
```

One of our primary goals is to align with other SDKs.  The golang SDK refers to this as "marker", rather than "initial_marker".

Reply:
done